app/assets/javascripts/govuk_publishing_components/components/cookie-banner.js

```diff
@@ -11,12 +11,16 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     this.$module.setCookieConsent = this.setCookieConsent.bind(this)
 
     this.$module.cookieBanner = document.querySelector('.gem-c-cookie-banner')
-    this.$module.cookieBannerConfirmationMessage = document.querySelector('.gem-c-cookie-banner__confirmation')
+    this.$module.cookieBannerConfirmationMessage = this.$module.querySelector('.gem-c-cookie-banner__confirmation')
 
     // Temporary check while we have 2 banners co-existing.
```

Comment:
Not really part of this PR, but I'm guessing this doesn't apply anymore?

Reply:
Yep, I'll try and do some clean up on those comments.